src/stacks-hierarchy/Root_TabNavigatorStack/Root_TabNavigatorStack.tsx

```diff
@@ -99,12 +104,21 @@ export const Root_TabNavigatorStack = ({navigation}: Props) => {
         );
       }
     }
+
+    const notificationsNotGranted =
+      pushNotificationPermissionStatus == 'undetermined' ||
+      pushNotificationPermissionStatus == 'denied';
```

Comment:
Do you know how this will work on Android? From what I understand, `'denied'` can mean both "we haven't asked the user yet" and "the user have actively denied permissions". Which might lead to us showing the onboarding more times that strictly necessary? But if that's true, that might be ok?

Reply:
I did read up about it quickly and tested it, but will double check it

Reply:
the undetermined was not necessary and the [denied handles it for both iOS and android ](https://github.com/zoontek/react-native-permissions#understanding-permission-flow)

Reply:
I don't think removing 'undetermined' makes sense here, since that is the initial state on iOS after a new install. Maybe a good idea to have a flow chart like that for the app, but I don't think this one is correct for notifications, since we use Firebase Messaging instead of react-native-permissions. 

If I understand the Android behavior correctly, I think this should look like:

```
Platform.OS === 'ios' 
	? pushNotificationPermissionStatus === 'undetermined'
	: pushNotificationPermissionStatus === 'denied'
```

Reply:
This clarifies, I for some reason believed it was handled by the same permissions library. Will look into it, and look at the right place now